### ISSUE TITLE
terraform-config-inspect command error

### ISSUE THREAD

**Issue by mkm-r** [IBM-Cloud/terraform-config-inspect#8]
Noticed below error while running `terraform-config-inspect` command for one of the GE module.
```
terraform-config-inspect --json --metadata ../terraform-provider-ibm/metadata/provider_metadata.json
2022/08/18 12:08:55 loadModuleErr: Failed to load modules LoadIBMModuleErr: Error while loading child modules LoadModuleFromFilesystemErr: Failed to read module directory: Module directory ./.terraform/modules/zone_subnet_addrs/github.com/hashicorp/terraform-cidr-subnets.git?ref=v1.0.0 does not exist or cannot be read. (and 1 other messages)
```

**Comment by ocofaigh** [comment]
@mkm-r Any progress on this?

**Comment by mkm-r** [comment]
@ocofaigh we are releasing the fix by this week. @kavya498 FYI

**Comment by kavya498** [comment]
@ocofaigh , I have fixed the issue and merged..
Meanwhile I m testing all possible scenarios..

**Comment by kavya498** [comment]
This fix has been released in 1.0.0-beta1
